### PR TITLE
Added documentation for .claspignore

### PR DESCRIPTION
**Pull request by thesiti92**
I also included a sample succinct .claspignore file for a large npm project to save users from overly verbose ignore files.

### CODE REVIEW

**Comment by thesiti92** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -129,4 +129,19 @@ Bump versions and publish with:
...
+As of now, clasp does not support pushing only tracked files, so to push a few files from an npm project with bundling, for example, a sample `.claspignore` could look like:
+
+```
+**/**
+!build/Main.gs
+!appsscript.json
+```  
+This file ignores everything but the manifest and the bundle.
+
+Note: the `.claspignore` file is parsed with [Anymatch](https://github.com/micromatch/anymatch), making it match files differently from a typical `.gitignore`, especially with directories. To ignore a directory, use syntax like: `**/node_modules/**`
```
Comment (grant) [inline README.md]: > "use syntax like: `**/node_modules/**`"

You already have a colon for "Note:". Just say:

use syntax like `**/node_modules/**`.

File: README.md [hunk 2]
```diff
@@ -129,4 +129,19 @@ Bump versions and publish with:
 npm publish --access public
 ```
 
+## Ignore Files
+
+Create a file called `.claspignore` in the root directory of your Apps Script project. Add files as if it were a .gitignore, and they will be excluded from `clasp push`, `clasp deploy`, etc...  
+
+As of now, clasp does not support pushing only tracked files, so to push a few files from an npm project with bundling, for example, a sample `.claspignore` could look like:
```
Comment (grant) [inline README.md]: > As of now, clasp does not support pushing only tracked files, so to push a few files from an npm project with bundling, for example, a sample `.claspignore` could look like:

I don't think `clasp` will ever support tracked files like git. Let's just say:

"""
A sample `.claspignore` could look like:
"""

The next sentence explains the file.

File: README.md [hunk 3]
```diff
@@ -129,4 +129,19 @@ Bump versions and publish with:
 npm publish --access public
 ```
 
+## Ignore Files
+
+Create a file called `.claspignore` in the root directory of your Apps Script project. Add files as if it were a .gitignore, and they will be excluded from `clasp push`, `clasp deploy`, etc...
```
Comment (grant) [inline README.md]: > Add files as if it were a .gitignore

Add patterns as if it were a .gitignore

**Comment by thesiti92** [comment]
All changes should be there!

**Comment by grant** [comment]
LGTM!

**Pull request merged**